week3/1-Tuesday/class/exercise 6/stephen-answer.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Stephen Roach</title>
+</head>
+    <body>
+        <h1>This covers everything we have COVERED!</h1>
+        <hr>
+        <p><i>We are done for now,</i><b> so I believe Adam will speak about:</b></p>
+        <ul>
+            <li>Adding attributes to tags</li>
+            <li>And adding links and images to your websites</li>
+            <li>He'll also remind you that:</li>
+        </ul>
+        <ol>
+            <li><h1><b>You is KIND!</h1></li>
+            <li><h1><b>You is SMART!</h1></li>
+            <li><h1><b>You is IMPORTANT!</h1></li>
```

Comment:
Don't forget to close the bold tags here! You have to make sure balance those tags even thought it looks right.